src/subdomains/staking/domain/entities/reward-batch.entity.ts

```diff
@@ -97,7 +97,7 @@ export class RewardBatch extends IEntity {
 
```

Comment:
Mismatch is rounded by 8 digits and min. correction is 10^-8, whereas transactionsTotal is 16 digit precision. Does this make sense?